tools/rgrnastar/rg_rnaStar.xml

```diff
@@ -282,15 +282,22 @@
                     <param name="GTFselect" type="select"
                            label="Reference genome with or without an annotation"
                            help="Select the '... with builtin gene-model' option to select from the list of available indexes that were built with splice junction information. Select the '... without builtin gene-model' option to select from the list of available indexes without annotated splice junctions, and, optionally, provide your own splice-junction annonations.">
-                        <option value="without-gtf" selected='true'>use genome reference without builtin gene-model</option>
+                        <option value="without-gtf-with-gtf" selected='true'>use genome reference without builtin gene-model but provide a gtf</option>
```

Comment:
Here is the new mode: 'without-gtf-with-gtf' which is: the index was built without but I will provide a gtf at runtime.

Reply:
Thanks for the clarification. Overlooked it :)